meinberlin/assets/scss/_layout.scss

```diff
@@ -66,7 +66,7 @@
     }
 
     .l-tiles-3 {
-        @include grid-tiles(3);
+        @include grid-tiles(2);
```

Comment:
I guess this also helps for the other cases where we use the tiles!

Reply:
we don't use it